server/src/models/Route.js

```diff
@@ -94,13 +139,24 @@ class Route extends Model {
    * returns route context
    * @returns {{ max_distance: number, max_duration: number }}
    */
-  static async getFilterContext() {
+  static async getFilterContext(source) {
+    const { isMad } = source
```

Comment:
Can you move this destructuring into the arguments like the other methods do (or should be doing if I missed one)? And add JSdocs with the DbContext type, as above.

Reply:
I couldn't find any example of jsdocs with DbContext type, but I changed the parameter